Automattic/wp-calypso
# Enable running CoBlocks tests on dedicated edge site

#### Changes proposed in this Pull Request

* Update e2e config with dedicated `coblocks-edge` stickered blog creds
* Update CoBlocks tests to run for edge user when `COBLOCK_EDGE` env var is set

#### Testing instructions

This PR should have the `[Status] Needs e2e Testing CoBlocks Edge` label added that should trigger CoBlocks edge site tests:
- `ci/wp-e2e-tests-full-coblocks-edge-desktop` and
- `ci/wp-e2e-tests-full-coblocks-edge-mobile`.

These tests should also be passing :)

To make sure that the correct test account is being used you can go to one of the tests builds and check in the `Set e2e environment variables` section if the `COBLOCKS_EDGE` is set to `true`.

Related:
- https://github.com/Automattic/wp-e2e-tests-gh-bridge/pull/87
- https://github.com/Automattic/wp-e2e-tests-for-branches/pull/61
- https://github.com/Automattic/wp-e2e-tests-for-branches/pull/62
